catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx

```diff
@@ -77,6 +101,28 @@ const PhysicalItems: FunctionComponent<Props> = ({
     })
     .filter(Boolean);
 
+  useEffect(() => {
+    const addStatusToItems = async () => {
+      const work = await fetchWorkItems(workId);
+      if (!isCatalogueApiError(work)) {
+        const mergedItems = itemsRef.current.map(currentItem => {
+          const matchingItem = work.items?.find(
+            item => item.id === currentItem.id
+          );
+          return {
+            ...matchingItem,
+            ...currentItem,
+          };
+        });
+        setPhysicalItems(mergedItems);
+      }
+      // else {
+      // tell the user something about not being able to retrieve the status of the item(s)
+      // }
```

Comment:
Whatever this ends up being - we may find we run into `429`s from our rate limiting, so worth bearing in mind that we might want to handle that as a separate case